components/atoms/Card/card.tsx

```diff
@@ -4,16 +4,18 @@ interface CardProps {
   className?: string;
   heading?: ReactNode;
   children: ReactNode;
+  focusable?: boolean;
 }
 
-const Card: React.FC<CardProps> = ({ className, children, heading }) => {
+const Card = ({ className, children, heading, focusable = false }: CardProps) => {
```

Comment:
I didn't make it focusable by default because I wasn't sure if that's necessary or not.